moveit_planners/ompl/ompl_interface/src/constraints_library.cpp

```diff
@@ -461,7 +461,7 @@ ompl::base::StateStoragePtr ompl_interface::ConstraintsLibrary::constructConstra
 
   // construct the constrained states
 
-  robot_state::RobotState robot_state(default_state);
+  robot_state::RobotState robot_state(default_state);  // NOLINT(performance-unnecessary-copy-initialization)
```

Comment:
Does this not compile after linting, or does it just add a bug implicitly? You're correct; this shouldn't be `const`. I just didn't think that `clang-tidy` would put the project in a state where it can't compile.